test/e2e/tests/metrics/state-snapshots/errors-after-init-opt-in-background-state.json

```diff
@@ -38,15 +38,15 @@
     "showNetworkBanner": true,
```

Comment:
the changes of this file are the result of running the test with the update-snapshot flag, so we make sure that the state includes the changes related to being logged in

`yarn test:e2e:single test/e2e/tests/metrics/errors.spec.js --browser=chrome --update-snapshot`